src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolverVariableInjectorInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\ArgumentResolver;
+
+/**
+ * return a key named array for adding variables to ExpressionLanguage in EntityResolver
+ *
+ * @author Roman JOLY <eltharin18@outlook.fr>
+ */
+interface EntityValueResolverVariableInjectorInterface
+{
+    public function getVariables() : array;
```

Comment:
If you know that the array wil only have names as key, you may add some PHPDoc to better hint the return type? Like `array<string, mixed>` or whatever is relevant here